src/SFA.DAS.ApprenticeCommitments.UnitTests/Application/Services/CommitmentsV2ServiceTests.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Threading.Tasks;
+using AutoFixture.NUnit3;
+using FluentAssertions;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.ApprenticeCommitments.Application.Services;
+using SFA.DAS.ApprenticeCommitments.Configuration;
+using SFA.DAS.SharedOuterApi.Infrastructure;
+using SFA.DAS.SharedOuterApi.Interfaces;
+
+namespace SFA.DAS.ApprenticeCommitments.UnitTests.Application.Services
+{
+    [TestFixture]
+    public class CommitmentsV2ServiceTests
+    {
+        [Test, AutoData]
+        public async Task WhenApprenticeshipIsNotFound(
+            long accountId,
+            long apprenticeshipId,
+            [Frozen] Mock<IInternalApiClient<CommitmentsV2Configuration>> client)
+        {
+            var sut = new CommitmentsV2Service(client.Object);
+            Func<Task> func = async () => await sut.GetApprenticeshipDetails(accountId, apprenticeshipId);
+            
+            func.Should().Throw<HttpRequestContentException>();
```

Comment:
There is an alternative syntax which may (or may not!) be appealing

```suggestion
            sut.Invoking(s => s.GetApprenticeshipDetails(accountId, apprenticeshipId))
                .Should().Throw<HttpRequestContentException>();

```

Reply:
I prefer your suggestion, easier to read and makes the intent clearer